README.md

```diff
@@ -37,6 +39,7 @@ Setup database with:
    rails db:migrate
 ```
 
+## Live link
```

Comment:
- What does TBA mean? Is it an instruction for deploying the app to heroku? Please give us precise details on how one can deploy this app to heroku